src/stream/src/executor/backfill/utils.rs

```diff
@@ -336,29 +337,28 @@ pub(crate) fn mark_cdc_chunk(
 /// For each row of the chunk, forward it to downstream if its pk <= `current_pos` for the
```

Comment:
`pk <= current_pos` -> `dist_key <= current_pos`?

Reply:
Nope, this condition doesn't change. Only when computing `vnode`, we need to use the proper function.